thrift/thrift0.13/src/main/java/com/linecorp/armeria/internal/server/thrift/ThriftDocServicePlugin.java

```diff
@@ -124,7 +126,7 @@ public ServiceSpecification generateSpecification(Set<ServiceConfig> serviceConf
                     final Class<?> serviceClass = iface.getEnclosingClass();
                     final EntryBuilder builder =
                             map.computeIfAbsent(serviceClass, cls -> new EntryBuilder(serviceClass));
-
+                    builder.thriftServiceEntry(entry);
```

Comment:
Given that `entry` is associated with `serviceClass` and it is determined when `EntryBuilder` is created, it would be better to set it in the constructor.
```java
map.computeIfAbsent(serviceClass, cls -> new EntryBuilder(serviceClass, entry));
```

Reply:
Got it, should I add a new constructor for this?
or change `public EntryBuilder(Class<?> serviceType)` to `public EntryBuilder(Class<?> serviceType, ThriftServiceEntry thriftServiceEntry)`.

If change the constructor, need change some tests  code, for example,  `new EntryBuilder(FooService.class)` ->  `new EntryBuilder(FooService.class, null)`.
I'm not feel good for this change.